frontend/src/app/forms/steps/field-designation-page/field-designation-page.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="main-description">
+  <h1>Industries and Positions</h1>
+  <span>
+    So that we can know in which industries and positions you are intrested,
+    please choose from the following.
+  </span>
+</div>
+
+<ion-item class="ion-no-padding">
+  <ion-label color="medium">Industry</ion-label>
+  <ion-select
+    multiple
+    [formControl]="formsData.controls.fieldDesignationInfo.controls.field"
+    [interfaceOptions]="{ cssClass: 'custom-popover' }"
+    interface="popover"
```

Comment:
On the browser's console, you will see the warning 
```
Select interface cannot be "popover" with a multi-value select. Using the "alert" interface instead.
```
So it would be better to remove `interface="popover"`. Also, maybe rename the css class names like "custom-popover" (since this is not a popover anymore :) )

Reply:
Just leave without interface which will be alert by default.

Reply:
Okay i will leave this.